rotkehlchen/tests/external_apis/test_monerium.py

```diff
@@ -192,3 +196,61 @@ def test_bridge_via_monerium(task_manager, database, monerium_credentials):  # p
             has_premium=True,
         )
     assert new_events == [gnosis_event, eth_event]
+
+
+@pytest.mark.parametrize('default_mock_price_value', [ONE])
+def test_query_info_on_redecode_request(rotkehlchen_api_server: APIServer):
+    """Test that triggering a re-decode for a monerium transaction updates correctly the notes"""
+    database = rotkehlchen_api_server.rest_api.rotkehlchen.data.db
+    dbevents = DBHistoryEvents(database)
+    amount_str = '1500'
+    gnosishash = deserialize_evm_tx_hash(val='0x10d953610921f39d9d20722082077e03ec8db8d9c75e4b301d0d552119fd0354')  # noqa: E501
```

Comment:
nitpick: Please compact initializations in tests
```suggestion
    database, dbevents, amount_str, gnosis_hash = rotkehlchen_api_server.rest_api.rotkehlchen.data.db, DBHistoryEvents(database), '1500', deserialize_evm_tx_hash(val='0x10d953610921f39d9d20722082077e03ec8db8d9c75e4b301d0d552119fd0354')  # noqa: E501
```